src/component.jl

```diff
@@ -147,7 +161,12 @@ function simulate_component(rng, c::LinearModelComponent, design::AbstractDesign
     return y' .* c.basis
 end
 """
-simulate MixedModelComponent
+    simulate_component(rng, c::MixedModelComponent, design::AbstractDesign)
+Generates a MixedModel and simulates data according to c.β and c.σs
+
+A trick is used to remove the Normal-Noise from the MixedModel which might lead to rare numerical instabilities. Practically, we upscale the σs by factor 10000, and provide a σ=0.0001. Internally this results in a normalization where the response scale is 10000 times larger than the noise.
+
+Currently it is not possible to use a different basis for fixed and random effects, but a code-stub exists (it is slow though)
 
 julia> design = MultiSubjectDesign(;n_subjects=2,n_items=50,item_between=(;:cond=>["A","B"]))
```

Comment:
I would suggest putting the code in a docstring test. Moreover, it should be `items_between`.

Reply:
see above about docstring test, the `s` have been added thanks!